### PR TITLE
Add s3_bucket_access_allowed_all query Closes #1554

### LINKED ISSUE #1554: Add s3_bucket_access_allowed_all query for Ansible
**Platform**
Ansible

**Provider**
AWS

**Description**
Check if the s3 bucket is accessible by all AWS account users

### PR DESCRIPTION
**Pull request by NunoAraujoCX**
Closes #1554 
Check if the s3 bucket is accessible to all users

### CODE REVIEW

File: assets/queries/ansible/aws/sqs_queue_exposed/test/negative.yaml [hunk 1]
```diff
@@ -7,4 +7,20 @@
...
+                     "Effect":  "Allow",
+                     "Action": "sqs:SendMessage",
+                     "Resource": "${aws_sqs_queue.q.arn}",
+                     "Condition": {
+                       "ArnEquals": {
+                         "aws:SourceArn": "${aws_sns_topic.example.arn}"
+                       }
+                     }
+                  }
+                 ] }'
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/sqs_queue_exposed/test/negative.yaml:26]: why was this changed? A JSON dictionary is not  in JSON is YAML

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]

**Comment by markmishaevcx** [comment]
Scan submitted to Checkmarx

**Review by joaoReigota1 — APPROVED** [review]

**Comment by markmishaevcx** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 4 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1 Low**<br />Info&nbsp;**0 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049134&projectid=754)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[30](https://github.com/Checkmarx/kics/blob/feature/ansible_s3_bucket_access_allowed_all/cmd/builder/main.go#L30) |Medium|Path_Traversal|cmd/builder/main.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049134&projectid=754&pathid=3)
[52](https://github.com/Checkmarx/kics/blob/feature/ansible_s3_bucket_access_allowed_all/pkg/kics/service.go#L52) |Medium|Denial_Of_Service_Resource_Exhaustion|pkg/kics/service.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049134&projectid=754&pathid=1)
[83](https://github.com/Checkmarx/kics/blob/feature/ansible_s3_bucket_access_allowed_all/test/main_test.go#L83) |Medium|Denial_Of_Service_Resource_Exhaustion|test/main_test.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049134&projectid=754&pathid=2)
